index.js

```diff
@@ -1,13 +1,43 @@
+function buildStylesheets (sheets, async) {
+  var output = ''
+  if (!sheets) return output
+
+  if (typeof sheets === 'string') {
+    sheets = [sheets]
+  }
+
+  sheets.forEach(function (sheet) {
+    output += !async
+      ? `<link rel="stylesheet" href="${sheet}">`
+      : `<link rel="stylesheet" href="${sheet}" media="none" onload="if(media!=='all')media='all'">`
+  })
+
+  return output
+}
+
+function buildScripts (scripts, async) {
+  var output = ''
+  if (!scripts) return output
+
+  if (typeof scripts === 'string') {
+    scripts = [scripts]
+  }
+
+  scripts.forEach(function (script) {
+    output += !async
```

Comment:
What do you think about adding a new line between `<script>` and `<link>` tags?

That would make the end result a little easier to look at if a user has to go back and make changes later (I'm thinking of the cli use case here).